frontend/src/composables/useRequestTransaction.ts

```diff
@@ -122,8 +122,8 @@ export function useWaitRequestFilled(beamerConfig: Ref<Readonly<BeamerConfig>>)
       await waitOnFulfillment();
       requestState.value = RequestState.RequestSuccessful;
     } catch (error) {
-      console.log(error);
```

Comment:
I think it is helpful to log the full error, but maybe not here and instead where it is actually caught. And `console.error` should rather be used.

Reply:
I think it will be logged anyway in the console if it's a Metamask error (which is the current case here)!